docs/Environment Variables.md

```diff
@@ -0,0 +1,109 @@
+# Environment variables
+
+## Overridable variables
+
+| Name                            | Description |
+| ------------------------------- | ----------- |
+| XDG_DATA_HOME                   | If set, uses `$XDG_DATA_HOME/nerves` as the data directory. Defaults to `~/.nerves` if unset |
+| NERVES_DL_DIR                   | Path where compressed Nerves system and toolchain artifacts are downloaded during `mix deps.get`. Defaults to `$XDG_DATA_HOME/nerves/dl` |
+| NERVES_ARTIFACTS_DIR            | Path where Nerves system and toolchain artifacts are decompressed (from `$NERVES_DL_DIR/<artifact-name>`) and cached for use with compilation. Defaults to `$XDG_DATA_HOME/nerves/artifacts` |
+| NERVES_ENV_DISABLED             | Set to `1` to disable the nerves_package compiler |
+| NERVES_DEBUG                    | Set to `1` to print out debug info during compilation |
+| NERVES_LOG_DISABLE_PROGRESS_BAR | Set to `1` to disable progress bar output when fetching artifacts (typically for CI) |
+| SOURCE_DATE_EPOCH               | Used for [reproducable builds](https://reproducible-builds.org). Can also be set via `config :nerves, source_date_epoch: val` |
+
+## Set by Nerves tooling
+
+When compiling (Elixir or non-BEAM code), Nerves sets environment variables to
+guide compilation. These environment variables are available to `mix`, `rebar3`
+and any code invoked from them. For example, these are frequently used in the
+`Makefiles` invoked by [`elixir_make`](https://hex.pm/packages/elixir_make).
```

Comment:
```suggestion
## Nerves-provided environment variables

Nerves sets the environment variables in this section to control compilation. Most
variables affect the compilation of C and C++ code so that they use the right
crosscompiler, flags, and directories. These environment variables are available to 
`mix`, `rebar3` and any code invoked from them. For example, these are frequently
 used in the `Makefiles` invoked by [`elixir_make`](https://hex.pm/packages/elixir_make).
```